src/frontend/src/flows/manage/authenticatorsSection.ts

```diff
@@ -124,26 +127,33 @@ export const authenticatorItem = ({
     dupCount,
     warn,
     info,
-    remove,
+    device,
     rename,
     isCurrent,
+    canBeRemoved,
   },
   index,
   i18n,
   icon,
+  onRemoveDevice,
 }: {
   authenticator: DedupAuthenticator;
   index: number;
   i18n: I18n;
   icon?: TemplateResult;
+  onRemoveDevice: (device: DeviceData) => void;
```

Comment:
If the `device` is passed into `authenticatorItem` already as argument and there is only a single `device` passed in, there's no need for the `device` argument in `onRemoveDevice`.

I would suggest keeping the method abstract:

```suggestion
  onRemove: () => void;
```